docs/reference/commandline/service_create.md

```diff
@@ -474,6 +474,42 @@ accessible at the target port on every node regardless if there is a task for
 the service running on the node. For more information refer to
 [Use swarm mode routing mesh](https://docs.docker.com/engine/swarm/ingress/).
 
+### Templating
+
+Some flags of `service create` support the use of templating. Go's
+[text/template](http://golang.org/pkg/text/template/) package
+describes all the details of the format.
+
+The supported flags are the following :
+
+- `--hostname`
+- `--mount`
+- `--env`
+
+Valid placeholders for the Go template are listed below:
+
+Placeholder       | Description
+----------------- | --------------------------------------------
+`.Service.ID`     | Service ID
+`.Service.Name`   | Service name
+`.Service.Labels` | Service labels
+`.Node.ID`        | Node ID
+`.Task.ID`        | Task ID
+`.Task.Name`      | Task name
+`.Task.Slot`      | Task slot
+
+For example:
+
+```bash
+$ docker service create --name hosttempl --hostname="{{.Node.ID}}-{{.Service.Name}}" busybox top
```

Comment:
You'll need to wrap the double-backeted content with `{% raw %}` and `{% endraw %}`